src/AstrometryFit.cc

```diff
@@ -237,17 +124,17 @@ void AstrometryFit::LSDerivatives1(const CcdImage &ccdImage, TripletList &tList,
     // refraction stuff
     Point refractionVector = ccdImage.getRefractionVector();
     // transformation from sky to TP
-    const Gtransfo *sky2TP = _astrometryModel->getSky2TP(ccdImage);
+    const Gtransfo *sky2TP = _astrometryModel.getSky2TP(ccdImage);
     // reserve matrices once for all measurements
     GtransfoLin dypdy;
-    // the shape of h (et al) is required this way in order to be able to
+    // the shape of H (et al) is required this way in order to be able to
```

Comment:
Glad to see I'm not alone in using capital letters for variables when consistency with math-heavy documentation is what really matters.